docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md

```diff
@@ -0,0 +1,361 @@
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type in `10` and press `Enter`.
+
+**Expected result**
+
+The DUT should be turned off.
+
+## DTS004.001 DTS option reboot DUT works correctly
+
+**Test description**
+
+This test aims to verify that the option `Reboot system` in the DTS menu reboots
+the DUT.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Plug the USB stick with DTS into the USB slot on the DUT.
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type in `11` and press `Enter`.
+
+**Expected result**
+
+The DUT should be restarted.
+
+## DTS005.001 DTS drop-to-shell option works correctly
+
+**Test description**
+
+This test aims to verify that the option `Shell` in the DTS menu opens Shell.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Plug the USB stick with DTS into the USB slot on the DUT.
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type in `9` and press `Enter`.
+
+**Expected result**
+
+1. Information about entering the shell and how to exit should be displayed.
+1. Shell command input should be activated.
+
+Example output:
+
+```bash
+Entering shell, to leave type exit and press Enter or press LCtrl+D
+
+bash-5.1#
+```
+
+## DTS006.001 Flash device from DTS shell by using flashrom works correctly
+
+**Test description**
+
+This test aims to verify whether is the possibility to flash the DUT firmware by
+using flashrom in `DTS Shell`.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. `{PATH}` = individual path to a specific binary.
+
+**Test steps**
+
+1. Plug the USB stick with DTS into the USB slot on the DUT.
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type 9 and click `Enter` to launch Shell.
+1. Run the following command to obtain `coreboot.rom` binary:
+
+    ```bash
+    wget https://3mdeb.com/open-source-firmware/{PATH} -O /tmp/coreboot.rom
+    ```
+
+    > This is not only way to obtain binary. For example you can use `scp`.
+
+1. Run the following command to flash the firmware:
+
+    ```bash
+    flashrom -p internal -w /tmp/coreboot.rom
```

Comment:
Are You sure about this command? We are going to flash all of the flash chip?

Reply:
I added information on where to find the exact command for each platform.